python/samples/getting_started_with_processes/step01_processes.py

```diff
@@ -0,0 +1,210 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+from typing import Generic, TypeVar
+
+from semantic_kernel import Kernel
+from semantic_kernel.connectors.ai.chat_completion_client_base import ChatCompletionClientBase
+from semantic_kernel.connectors.ai.open_ai import OpenAIChatCompletion
+from semantic_kernel.contents import ChatHistory
+from semantic_kernel.functions import kernel_function
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+from semantic_kernel.processes.kernel_process.kernel_process_step_context import KernelProcessStepContext
+from semantic_kernel.processes.kernel_process.kernel_process_step_state import KernelProcessStepState
+from semantic_kernel.processes.local_runtime.local_event import KernelProcessEvent
+from semantic_kernel.processes.process_builder import ProcessBuilder
+from semantic_kernel.processes.process_function_target_builder import ProcessFunctionTargetBuilder
+
+
+class CommonEvents:
+    UserInputReceived = "UserInputReceived"
+    AssistantResponseGenerated = "AssistantResponseGenerated"
+
+
+class ChatBotEvents:
+    StartProcess = "startProcess"
+    IntroComplete = "introComplete"
+    AssistantResponseGenerated = "assistantResponseGenerated"
+    Exit = "exit"
```

Comment:
should events be string enums in python? In c# we had to use static classes because there is no concept of string of enums in dotnet.

The idea behind event names is to try to ensure all event names(event strings) are unique

Reply:
I agree, mentioned this in the other sample as well, a Enum would be better!

Reply:
Updated to use enums. Thanks.